File: demo/src/test/java/com/example/demo/RentalTest.java
package com.example.demo;


import com.example.demo.entity.Rental;
import com.example.demo.repository.RentalRepository;
import com.example.demo.service.RentalService;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.transaction.annotation.Transactional;

import java.time.LocalDate;

@SpringBootTest
public class RentalTest {
    @Autowired
    RentalService rentalService;

    @Test
    void insertTest(){

//        Rental rental = Rental.createRental
//        rental.setId(1L);
//        rental.setBookId(1L);
//        rental.setUserId(1L);
//        rental.setRentDate(LocalDate.now());
//        rentalService.insertRental(rental);

    }
}

File: testproj/src/test/kotlin/com/example/testproj/testproj/RentalTest.kt
package com.example.testproj.testproj

import com.example.testproj.testproj.entity.Rental
import com.example.testproj.testproj.service.RentalService
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
internal class RentalTest @Autowired constructor(
        val rentalService: RentalService
){
    @Test
    internal fun getRentalByIdTest(){

    }

    @Test
    internal fun getRentalTest(){

    }

    @Test
    internal fun setRentalTest(){
        //
    }
}